### PR TITLE
Include github issues in the total issues list

### PR DESCRIPTION
**Pull request by TheRealFalcon**
also fixing some minor bashisms in the area

### CODE REVIEW

**Review by aciba90 — APPROVED** [review]
LGTM, thanks!

**Review by a-dubs — APPROVED** [review]

File: scripts/upstream-release [hunk 1]
```diff
@@ -28,9 +28,11 @@ git pull
 
 
 RELEASE_TIME_SPAN=${RELEASE_TIME_SPAN/ ago/}
-NUM_CONTRIBUTORS=`git log $PREVIOUS_MAJOR_MINOR..HEAD --pretty="%aN" | sort -u | wc -l`
-NUM_BUGS=`git log $PREVIOUS_MAJOR_MINOR..HEAD | log2dch | grep "LP: #" | wc -l`
-CHANGELOG=`git log $PREVIOUS_MAJOR_MINOR..HEAD | log2dch |  sed 's/^   //g'`
+NUM_CONTRIBUTORS=$(git log "$PREVIOUS_MAJOR_MINOR"..HEAD --pretty="%aN" | sort -u | wc -l)
+NUM_GITHUB_BUGS=$(git log "$PREVIOUS_MAJOR_MINOR"..HEAD | grep -c "Fixes GH")
```
Comment (holmanb) [inline scripts/upstream-release:32]: I think I recall a couple of `Fixes: GH` in commits that this would miss?

Comment (TheRealFalcon) [reply scripts/upstream-release:32]: I think I'm ok with that. I'm sure there are places that people have typed "LP #..." and we missed it due to the missed colon. The template says to use "Fixes GH-" and GH requires it for autoclosing, so I don't think that our tools should try to anticipate every possible way somebody is going to mistype things.

Comment (blackboxsw) [reply scripts/upstream-release:32]: Looks good, and we've already crossed the release boundary where we have no more missed `Fixes: GH` in footers.

Comment (blackboxsw) [reply scripts/upstream-release:32]: The last committer who did that was your's truly last  month, and I'm sure I'll never do that again ;)

**Pull request merged**